sysid/swerveConfig.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Is this a WIP (Work in progress)? If so, we should probably move it out of this PR and add it when it is ready in a later one.

Reply:
It's a completed sysID configuration for the drivebase